homeassistant/components/lutron/entity.py

```diff
@@ -27,10 +27,14 @@ async def async_added_to_hass(self) -> None:
         """Register callbacks."""
         self._lutron_device.subscribe(self._update_callback, None)
 
+    def _update_attrs(self) -> None:
+        """Update the entity's attributes."""
+
     def _update_callback(
         self, _device: LutronEntity, _context: None, _event: LutronEvent, _params: dict
     ) -> None:
         """Run when invoked by pylutron when the device state changes."""
+        self._update_attrs()
         self.schedule_update_ha_state()
```

Comment:
Should this just become a write state instead?

Reply:
No, since this method isn't async safe.